example/full-config.php

```diff
@@ -193,6 +196,20 @@
 
             DependencyFactory::class => DependencyFactoryFactory::class,
             ConfigurationLoader::class => ConfigurationLoaderFactory::class,
+
+            SchemaAssetsFilter::class => static function (): callable {
+                /**
+                 * @param AbstractAsset|string $asset
+                 */
+                return static fn (mixed $asset): bool => ! in_array(
+                    $asset instanceof AbstractAsset ? $asset->getName() : $asset,
+                    [
+                        'sequence_to_generate_value',
+                        'table_without_doctrine_mapping',
+                    ],
+                    true,
+                );
```

Comment:
Let's expand the comment (since this is reference documentation) explaining that this is mostly filtering out the two tables that are listed in the closure